README.md

```diff
@@ -67,7 +67,7 @@ quickstart();
 
 ## Samples
 
-Samples are in the [`samples/`](https://github.com/googleapis/nodejs-local-auth/tree/main/samples) directory. Each sample's `README.md` has instructions for running its sample.
+Samples are in the [`samples/`](https://github.com/googleapis/nodejs-local-auth/tree/master/samples) directory. Each sample's `README.md` has instructions for running its sample.
```

Comment:
this one is slightly more of a pain in the neck, and will need to be fixed when we move everything over.